weekly-openmeteo/extract.py

```diff
@@ -0,0 +1,41 @@
+'''Extracts weekly sunset and sunrise data for the 12 UK regions'''
+import logging
+from requests import get
+from api_error import APIError
+
+URL = "https://api.open-meteo.com/v1/forecast"
+PARAMS = {"latitude": [54.61, 56.49, 52.13, 54.97, 53.78, 53.8,
+                       52.9, 52.48, 52.24, 51.51, 51.28, 50.78],
+	      "longitude": [-6.62, -4.2, -3.78, -1.61, -2.7, -1.54,
+                       -1.23, -1.89, 0.9, -0.13, -0.78, -3.79],
+	      "daily": ["sunrise", "sunset"],"timezone": "Europe/London"}
```

Comment:
This could be for counties rather than regions so we get more precise times for a region as large as Scotland

Reply:
Good spot I will make these changes